pkg/controller/volume/pvcprotection/pvc_protection_controller_test.go

```diff
@@ -139,6 +142,8 @@ func generateUpdateErrorFunc(t *testing.T, failures int) clienttesting.ReactionF
 }
 
 func TestPVCProtectionController(t *testing.T) {
+	// unit tests assume that the PVC Protection feature is enabled
```

Comment:
setting global feature gates in unit tests is racy... consider passing a feature gate to the controller and checking the local one rather than mutating global state here (and leaving it mutated after the unit test runs).

See https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/node/node_authorizer.go#L53 as an example